tests/resources/test_conformance.py

```diff
@@ -54,6 +54,12 @@ async def test_landing_page_links(
     assert link.get("type") == expected_media_type
 
     link_path = urllib.parse.urlsplit(link.get("href")).path
+
+    # this feels hacky
+    # assert link_path == (
+    #     app.state.router_prefix + app.state.router_prefix + expected_path
+    # ) or (app.state.router_prefix + expected_path)
+
     assert link_path == app.state.router_prefix + expected_path
 
     resp = await app_client.get(link_path.rsplit("/", 1)[-1])
```

Comment:
ohhhh I know what's going on 
in 
https://github.com/jonhealy1/stac-fastapi-pgstac/blob/5d8e8fece8790e9cf070dd268a514b9db772de2d/tests/conftest.py#L165-L170

we should just have a base url be `http://stac.io` (don't had prefix) 

I'll fix this (because I think there is another issue 😓 )

Reply:
😬 I forgot to change https://github.com/stac-utils/stac-fastapi/blob/main/stac_fastapi/types/stac_fastapi/types/core.py#L608-L627 🤦